src/core/public/core_app/styles/_globals_v8dark.scss

```diff
@@ -2,6 +2,6 @@
 // ---
 // prepended to all .scss imports (from JS, when v8dark theme selected)
 
-@import "@elastic/eui/src/themes/eui-amsterdam/eui_amsterdam_colors_dark";
-@import "@elastic/eui/src/themes/eui-amsterdam/eui_amsterdam_globals";
+@import "@elastic/eui/src/themes/eui-next/eui_next_colors_dark";
```

Comment:
are we keeping the file name the same?

Reply:
the v8dark one? I figured if a plugin is using them, we don't want to break it; i would have loved to name it `next`.